backend/kesaseteli/applications/api/v1/views.py

```diff
@@ -21,14 +23,33 @@
 from applications.api.v1.serializers import (
     ApplicationSerializer,
     AttachmentSerializer,
+    SchoolSerializer,
     SummerVoucherSerializer,
     YouthApplicationSerializer,
 )
 from applications.enums import ApplicationStatus
-from applications.models import Application, SummerVoucher, YouthApplication
+from applications.models import Application, School, SummerVoucher, YouthApplication
 from common.utils import DenyAll
 
 
+class SchoolListView(ListAPIView):
+    # PostgreSQL specific functionality:
+    # - Custom sorter for name field to ensure finnish language sorting order.
+    # - NOTE: This can be removed if the database is made to use collation fi_FI.UTF8
+    _name_fi = Func(
```

Comment:
Nice one!

Reply:
Thanks! Adapted from this [Stackoverflow](https://stackoverflow.com/a/36423340) answer.